content/_data/changelogs/weekly.yml

```diff
@@ -12286,6 +12286,83 @@
     # pull: 5661 (PR title: Add missing `@since`)
     # pull: 5667 (PR title: Increase timeout for branch)
 
+  - version: '2.308'
+    date: 2021-08-24
+    changes:
+      - type: TODO
```

Comment:
I know this is already merged, but I have doubts regarding this so wanted to put them here for future reference,

- Shouldn't this have type: rfe?
- and listed last? since it has a 'developer' label
- and content of 'pr_title' be copied to 'message'?

Reply:
it needs some fixups if you want to send a PR to clean it up?

Reply:
I'll give it a try, thanks!
Also, while we're on it, can you explain the need for the "pr_title" tag?

Reply:
It just comes from the script doesn’t do anything

Reply:
Hi @timja, I tried to run the weekly changelog automation script from jenkinsci/jenkins using the command: 
_$ ../core-changelog-generator/generate-weekly-changelog.sh_,

But it didn't write any changelogs in my YAML file, and the script ended with the message: 

_remote: Invalid username & password.
fatal: Authentication failed for 'https://nobody:@github.com/jenkins-infra/jenkins.io.git_

Question: Do I need to have some specific privileges and/or AUTH_TOKEN with me that might solve this issue?

Reply:
You need to set these variables: https://github.com/jenkinsci/jenkins/blob/master/.github/workflows/changelog.yml#L52

it will automatically create a pull request though if you don’t comment out any lines in the script

Reply:
Thank you, I'll try this.